packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -142,6 +143,7 @@ $linkRightPadding: 20px;
 
   &.compositeLinkIsSelected {
     & a,
```

Comment:
any button or any anchor is way too specific. What about immediate descendant?

Reply:
or better, a specific class target.

Reply:
I should be able to assing these properties to the `link` class now, which is applied to all `NavLinks`